src/UserBundle/Controller/UserController.php

```diff
@@ -398,6 +398,40 @@ public function updateAction(Request $request, User $user)
         return new Response($json);
     }
 
+
+        /**
+     * Edit a user {id} with data in the body
+     *
+     * @Rest\Post("/users/{id}/updateLanguage", name="update_user")
```

Comment:
No uppercase or "-" in route. Also, you don't need to specify the action in the route, since POST already give that information. Maybe just /users/{id}/language